### DISCUSSION TITLE
Educational "Ethical Hacking" Manual

### DISCUSSION THREAD

**Discussion by APT256** [whatotter/pwnhyve#44] [General]
So, I have a couple of sons who are really into computers and electronics. Some of their friends are as well. They love the idea of having something like this that they can fit into their pockets. I have talked to them about being responsible with these types of tools and how they can get into trouble if they are not responsible. The issue I have is that they want to know how to do x,y,and z but they don't don't want to hear it from Dad. That being said, and thought into creating an education manual that walks new users through setting up and executing the various attacks that this little device is capable of doing? I am thinking that this would be a great addition to a High School IT program? I am not sure if the schools are dipping their toes into "Ethical Hacking" but personally I would much rather the students get exposed to their than have the police knocking on their parent's door.

**Comment by whatotter** [comment]
this is a great idea, once i'm done with the whole rename that's coming up (should hopefully be the last rename aswell, people struggle to say pwnhyve), i can make docs here on how to execute attacks and how they work, aswell as videos and stuff seeing them in action

i'm very glad that your sons and their friend's are interested in cybersecurity, hopefully they stay in the field 🎉

speaking about a highschool IT program, they're usually scared to teach cybersecurity because of the same old reason "what if they turn on us and hack us instead?", in which i say "then improve your security" 😆
